froide/foirequest/templatetags/foirequest_tags.py

```diff
@@ -151,6 +153,22 @@ def redact_message(message, request):
     return content
 
 
+@register.simple_tag
+def redact_request_description(foirequest, request):
```

Comment:
I would suggest adding return types here e.g. either `str` or `SafeString` so it's clear if the string has been escaped to avoid double escaping when the code is reused.